packages/parser/src/utils/get-media-query.ts

```diff
@@ -21,7 +21,12 @@ export function getMediaQuery(breakpoints?: Dict, mapper = toMediaQuery) {
 
   const asArray = objectKeys(_breakpoints)
     .map((key) => _breakpoints[key])
-    .sort((a, b) => a - b)
+    .sort((a, b) => {
+      const regex = /^\d+/
```

Comment:
Agree with @ljosberinn, these changes are perfect if you at least define `regex` outside of the function call so it's only created once. I'm not entirely sure of the performance implications, but we might as well avoid it anyway!